.release-please-manifest.json

```diff
@@ -81,6 +81,8 @@
   "plugins/source/awspricing+FILLER": "0.0.0",
   "plugins/destination/duckdb": "5.3.0",
   "plugins/destination/duckdb+FILLER": "0.0.0",
+  "plugins/destination/motherduck": "0.0.0",
```

Comment:
The file to edit it this one https://github.com/cloudquery/cloudquery/blob/8192d6d99898692e9232d29d9ac71f5d2c6d8d4a/release-please-config.json (see https://github.com/cloudquery/cloudquery/blob/8192d6d99898692e9232d29d9ac71f5d2c6d8d4a/contributing/adding_a_new_plugin_to_cq_monorepo.md).

Also before releasing make sure to create it under our team https://cloud.cloudquery.io/teams/cloudquery/packages/create-plugin

Reply:
![logo](https://github.com/cloudquery/cloudquery/assets/223029/993129ee-b16a-4fdd-b485-91be6ca63839)
Done, looks like I don't have admin rights though. Here's the logo

Reply:
update - ok, created as preview + private for now.